### PR TITLE
Add application-local config to start app with mock server Info easily 

### PR DESCRIPTION
**Pull request by allenzph**
…tion with mock server info easily

## Summary

...

## Before

_Description_

Before

## After

_Description_

After

## Note

_Null_

### CODE REVIEW

File: backend/src/main/resources/application-local.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+server:
+  port: 4322
+  servlet:
+    context-path: /api/v1
+logging:
+  level:
+    root: INFO
```
Comment (guzhongren) [inline backend/src/main/resources/application-local.yml:7]: Debug is better for local

**Review by guzhongren — APPROVED** [review]
Good job

**Comment by guzhongren** [comment]
Better to add a trello /jira card

File: backend/src/main/resources/application-local.yml (hunk 1 above)
Comment (allenzph) [reply backend/src/main/resources/application-local.yml]: done

**Pull request merged**